openpype/hosts/fusion/hooks/pre_fusion_setup.py

```diff
@@ -16,19 +20,80 @@ class FusionPrelaunch(PreLaunchHook):
 
     """
     app_groups = ["fusion"]
+    PROFILE_NUMBER = 16
```

Comment:
What is this number?

Reply:
This is an odd thing. I guess BlackmagicDesign devs intended to make the `FUSION_PROFILE_DIR` to be version specific, so from the v16 they made it to be `FUSION16_PROFILE_DIR`. But then they abandoned this, and despite it is already version 18, still `FUSION16_PROFILE_DIR` is used. I added the variable in case they would break this in future version (they really could), so we could easily change the version or disable it.

Reply:
Maybe add small comment above it so any future devs know "why the hell"...